tests/schema/test_union.py

```diff
@@ -1,11 +1,12 @@
 import sys
 from dataclasses import dataclass
 from textwrap import dedent
-from typing import Generic, List, Optional, TypeVar, Union
+from typing import Annotated, Generic, List, Optional, TypeVar, Union
```

Comment:
ah, Annotated should be imported from typing_extensions

Reply:
I get this.wrong every single time 😅